test/test_helper.rb

```diff
@@ -25,6 +25,67 @@
 
 require 'sucker_punch/testing/inline'
 
+# Mock up vault client
+class VaultClient
+  def logical
+    @logical ||= Logical.new
+  end
+
+  def self.vault_response_object(data)
+    Response.new(data)
+  end
+
+  class Response
+    attr_accessor :lease_id, :lease_duration, :renewable, :data, :auth
+    def initialize(data)
+      self.lease_id = nil
```

Comment:
instance_values.symbolize_keys doesn't seem to export them as nil, unless they are set.